### PR TITLE
Python3 tim

### PR DESCRIPTION
**Pull request by MarvinT**
looks good to me... been shown to work on python 2.7

I feel like the first update is a bit unnecessary and you should just have everything return the figures. I don't see any reason to make it a flag given that they're just returning none right now...

### CODE REVIEW

**Comment by timsainb** [comment]
I just didn't want it to interfere with any current code people are running. I'll update to return figs for everything.

**Comment by MarvinT** [comment]
did you ever update the functions to return the figures?

**Comment by theilmbh** [comment]
What's the deal with this PR?

**Comment by theilmbh** [comment]
This is what is currently on Zog.  Can I just resolve these conflicts in favor of this branch and merge? @MarvinT @timsainb

File: behav/plotting.py [hunk 1]
```diff
@@ -74,9 +74,8 @@ def plot_performance_calendar(subj, data_to_analyze, disp_counts=False, vmins=(0
                         cmap=cmap, cbar=not disp_counts,
                         vmin=vmin, vmax=vmax)
         g.set_title(title)
-    idk_what_ppl_were_thinking = [x[1] for x in list(pivoted.keys())]
-    g.set_xticklabels(_date_labels(idk_what_ppl_were_thinking))
-    return f
+    g.set_xticklabels(_date_labels((pivoted.keys()).levels[1]));
+    if return_fig: return f
```
Comment (MarvinT) [inline behav/plotting.py]:
```suggestion
    return f
```

**Comment by MarvinT** [comment]
This should be good to go now. At least it runs all the plots in the notebook and the notebook is cleaned up so that it is run in order and doesn't have strange reloading things. Any objections? @theilmbh @timsainb

**Comment by theilmbh** [comment]
Give me 15 minutes, I'm grabbing tea. Then I'll let you know.

**Comment by theilmbh** [comment]
Should be fine.

File: behav/plotting.py [hunk 2]
```diff
@@ -74,8 +74,7 @@ def plot_performance_calendar(subj, data_to_analyze, disp_counts=False, vmins=(0
                         cmap=cmap, cbar=not disp_counts,
                         vmin=vmin, vmax=vmax)
         g.set_title(title)
-    idk_what_ppl_were_thinking = [x[1] for x in list(pivoted.keys())]
-    g.set_xticklabels(_date_labels(idk_what_ppl_were_thinking))
+    g.set_xticklabels(_date_labels((pivoted.keys()).levels[1]));
```
Comment (MarvinT) [inline behav/plotting.py]:
```suggestion
    idk_what_ppl_were_thinking = [x[1] for x in list(pivoted.keys())]
    g.set_xticklabels(_date_labels(idk_what_ppl_were_thinking))
```

**Pull request merged**